### ISSUE TITLE
TIVO info erased when enabling

### ISSUE THREAD

**Issue by rtteachr** [mackworth/cTiVo#396]
I'm using Catalina and the latest CTivo build. When I click enable on remote Tivos after entering the IP address, media code, etc all of that data immediately disappears. I try to enter it again, click enable, gone. Ideas?

**Comment by mackworth** [comment]
If it finds a duplicate with a local TiVo (e.g. you're setting up a remote for later use), it is supposed to disable one or the other, but it definitely shouldn't erase the data. Can you set Debug level to Verbose, and do it again (preferably not while a download is going on), and then post the log? (BTW, don't use email response or it will lose the attachment.)  it may give me a clue as to what's going on.

**Comment by rtteachr** [comment]
Not sure where the log goes to post it?

**Comment by mackworth** [comment]
First, compress the file in the Finder (right Click compress) to create a zip file. Then click on the "View it on Github" link to get back to this issue on the github website. Finally just drag the zip file onto the box at the bottom for a new entry.

**Comment by rtteachr** [comment]
Let me rephrase that, where do I find it in the Finder/Hard Drive to send it to you?

**Comment by mackworth** [comment]
Sorry. In cTiVo’s Help menu, there’s a “Show logs in Finder” command.

**Comment by rtteachr** [comment]
[com.cTiVo.cTiVo 2019-11-13--01-01-21-288.log.zip](https://github.com/mackworth/cTiVo/files/3838926/com.cTiVo.cTiVo.2019-11-13--01-01-21-288.log.zip)

**Comment by mackworth** [comment]
So, which version of the software is this? (for some reason it's not in this log).  If it's not 3.4.1beta, then try that, as I think this is a different symptom of the bug I fixed.

**Comment by rtteachr** [comment]
3.4.1 (1097)

**Comment by mackworth** [comment]
Thank you for the log. I've walked through the code with the breadcrumbs it provides, and cannot see HOW it is doing what it indeed seems to be doing.(Nor can I recreate it on my system). I'd like to give you a more instrumented version to walk through what's happening in finer detail. If you;re ok with that, let's switch

**Comment by mackworth** [comment]
(Sorry, hit close button by accident.)

Thank you for the log. I've walked through the code with the breadcrumbs it provides, and cannot see HOW it is doing what it indeed seems to be doing, nor can I recreate it on my system.. I'd like to give you a more instrumented version to walk through what's happening in finer detail. If you're ok with that, let's switch over to email; just send me a note at mackworth at aol dot com.

**Comment by mackworth** [comment]
THhis issue is fixed in beta3 (1106). Please let me know if anyone still has an issue!
